tests/e2e/ans/ans.test.ts

```diff
@@ -146,8 +146,8 @@ describe("ANS", () => {
       domainName = randomString();
       subdomainName = randomString();
 
-      alice = Account.generate();
-      bob = Account.generate();
+      alice = Signer.generate();
+      bob = Signer.generate();
```

Comment:
how come we can do if bob's type is `Account`...?

Reply:
`Signer` is a specialization of `Account`. It still has an associatd public key and address.
You can't really "generate" an account,  since it does not have an associated private key,
buth we can still leverage `Signer.generate`